packages/inter-protocol/test/psm/test-governedPsm.js

```diff
@@ -44,14 +44,74 @@ test('psm block offers w/Governance', async t => {
 
   t.deepEqual(['wantMinted'], await E(zoe).getOfferFilter(psm.instance));
 
-  const giveCentral = AmountMath.make(knutIssuer.brand, 1_000_000n);
+  const giveCentral = AmountMath.make(knutIssuerKit.brand, 1_000_000n);
 
   await t.throwsAsync(
     () =>
       E(zoe).offer(
         E(psm.psmPublicFacet).makeWantMintedInvitation(),
         harden({ give: { In: giveCentral } }),
-        harden({ In: knutIssuer.mint.mintPayment(giveCentral) }),
+        harden({ In: knutIssuerKit.mint.mintPayment(giveCentral) }),
+      ),
+    { message: 'not accepting offer with description "wantMinted"' },
+  );
+});
+
+test('psm block offers w/charter', async t => {
+  const electorateTerms = { committeeName: 'EnBancPanel', committeeSize: 3 };
+  const timer = buildManualTimer(t.log, 0n, { eventLoopIteration });
+
+  const { knutIssuerKit, zoe, psm, committeeCreator, psmCharterCreatorFacet } =
+    await setupPsm(t, electorateTerms, timer);
+
+  const knutBrand = knutIssuerKit.brand;
+
+  // onchain, the invitations get sent to the committee member via their
+  // registered deposit facets. We'll skip that here.
+  const invitations = await E(committeeCreator).getVoterInvitations();
+
+  const cmInvitation = await E(
+    psmCharterCreatorFacet,
+  ).makeCharterMemberInvitation();
+
+  const cmSeat = E(zoe).offer(cmInvitation);
+  const invitationMakers = await E(cmSeat).getOfferResult();
+
+  const offerFilterInvite = await E(invitationMakers).VoteOnPauseOffers();
+
+  const offerArgs = harden({
+    instance: await psm.instance,
+    deadline: 2n,
+    strings: ['wantMinted'],
+  });
+
+  const requestVoteSeat = E(zoe).offer(offerFilterInvite, {}, {}, offerArgs);
+  const offerResult = await E(requestVoteSeat).getOfferResult();
+  const { details } = offerResult;
+  const { positions, questionHandle } = await details;
+
+  const exerciseAndVote = invitation => {
+    const seat = E(zoe).offer(invitation);
+    const { voter } = E.get(E(seat).getOfferResult());
+
+    return E(voter).castBallotFor(questionHandle, [positions[0]]);
```

Comment:
I don't think this fits the smart wallet design. Making the offer has to cast the vote.

Reply:
That's correct. This is using the older interface. (This test doesn't have a smart wallet.)

The smart wallet approach is tested in `test-psm-integration.js`

Reply:
Fair imo